wazimap/geo.py

```diff
@@ -35,11 +35,43 @@ class GeoData(object):
     new class in your `settings.py`. Wazimap will then load that class and make
     it available as `wazimap.geo.geo_data`.
     """
+    _versions = None
+
     def __init__(self):
         self.geo_model = Geography
         self.setup_levels()
         self.setup_geometry()
 
+    def _setup_versions(self):
+        """ Find all the geography versions.
+        """
+        self._versions = [x['version'] for x in self.geo_model.objects.values('version').distinct().all()]
+        self._latest_version = sorted(self.versions)[-1]
```

Comment:
@longhotsummer if I'm not mistaken, we can't do this globally - we need to find the latest per geography otherwise disestablished munis aren't found (not falling back to latest for _that_ muni) e.g. https://wazimap.co.za/profiles/municipality-EC103/
Interestingly, I can't override it https://wazimap.co.za/profiles/municipality-EC103/?geo_verision=2011